source/neuropod/bindings/java/src/main/java/com/uber/neuropod/LibraryLoader.java

```diff
@@ -1,19 +1,143 @@
+    // C++ library names. If we use the Neruopod Java API from a jar file these files may be packed into the jar file.
+    private static final List<String> EMBEDDED_LIB_NAMES = Arrays.asList(
+                                                                // Neuropod
+                                                                "libneuropod.so",
+                                                                // Backend libaries name
+                                                                "libneuropod_tensorflow_backend.so",
+                                                                "libneuropod_torchscript_backend.so",
+                                                                "libneuropod_pythonbridge_backend.so",
+                                                                // TensorFlow libraries name
+                                                                "libtensorflow_framework.so",
+                                                                "libtensorflow.so",
+                                                                // TorchScript libraries linux name
+                                                                "libc10.so",
+                                                                "libcaffe2_detectron_ops.so",
+                                                                "libcaffe2_module_test_dynamic.so",
+                                                                "libcaffe2_observers.so",
+                                                                "libfbjni.so",
+                                                                "libgomp-7c85b1e2.so.1",
+                                                                "libpytorch_jni.so",
+                                                                "libshm.so",
+                                                                "libtorch_cpu.so",
+                                                                "libtorch_global_deps.so",
+                                                                "libtorch_python.so",
+                                                                "libtorch.so",
+                                                                // TorchScript libraries darwin name
+                                                                "libc10.dylib",
+                                                                "libcaffe2.dylib",
+                                                                "libmklml.dylib",
+                                                                "libtorch.1.dylib",
+                                                                "libtorch.dylib",
+                                                                "libiomp5.dylib"
+                                                                );
+    private static final List<String> BIN_NAMES = Arrays.asList("neuropod_multiprocess_worker");
+    private static final String JNI_NAME = "neuropod_jni";
```

Comment:
Isn't it libneuropod_jni or it doesn't need "lib" prefix here? If so, please leave comment to make it clear.

Reply:
Yeah, it does not need lib prefix here because when call System.loadLibrary java will auto complete the lib and so part based on the OS.